dist/css/style.css

```diff
@@ -867,6 +868,13 @@ h1 {
   height: 1.4rem;
 }
 
+  .btn--green {
+    background-color: #fff;
+    border: 1px solid #396df2;
+    border-radius: 8px;
+    margin-left: 20px;
+  }
+
 /* Skills */
 .about-self {
   width:  5rem;
```

Comment:
Great design team🤝

- [ ] As requested by the previous reviewer, kindly ensure your mobile popup is a replica of the design given. Your mobile popup should have a close button to close the popup window.
- [ ] Kindly ensure your mobile popup has all features just like in the Figma design given. Your design seems to be hiding the buttons and the `Tonic` feature at the top of your design.
- Your design
![Your Mobile popup](https://github.com/mohashyne/MSalyhu_portfolio/assets/63917725/11e1d6dd-6b47-4f07-afeb-4003e45f0738)
- Given design
![Given mobile popup](https://github.com/mohashyne/MSalyhu_portfolio/assets/63917725/c07c5558-62c2-41a7-918f-95b3abef5c4c)

Reply:
# Required Changes Implemented

Close button was fixed and the required changes has been implemented